releases/release-1.10/release_team.md

```diff
@@ -1,13 +1,13 @@
 | **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...** |
 | ------ | ------ | ------ |
-| Lead | | |
-| Secondary | | |
-| Features | | |
+| Lead | Jaice Singer DuMars ([@jdumars](https://github.com/jdumars)) | Steven Wade ([@swade1987](https://github.com/swade1987)) |
+| Features | Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)) | 	Weiwei Jiang ([@wjiangjay](https://github.com/wjiangjay)) |
 | CI Signal | | |
-| Test Infra | | |
+| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Benjamin Elder ([@BenTheElder](https://github.com/BenTheElder)) |
 | Bug Triage | | |
 | Branch Manager | | |
-| Docs | | |
-| Release Notes | | |
-| Marketing | | |
-| Patch Release Manager | | |
+| Docs | Jennifer Rondeau ([@Bradamant3](https://github.com/Bradamant3)) | Jessica Yao ([@abiogenesis-now](https://github.com/abiogenesis-now)) |
+| Release Notes | Nick Chase ([@nickchase](https://github.com/nickchase)) | |
+| Marketing | Caitlyn O'Connell | |
+| Patch Release Manager | Shyam JVS ([@shyamjvs](https://github.com/shyamjvs)) | |
+| Approval Notifier | 	k8s-ci-robot ([@k8s-ci-robot](https://github.com/k8s-ci-robot)) | |
```

Comment:
This one was a joke. You don't have to include it in the final list. :)

Reply:
Bots never get enough recognition!  ;)  It's staying.

Reply:
This guy should be recognized :)